astropy/convolution/convolve.py

```diff
@@ -384,7 +384,7 @@ def convolve_fft(array, kernel, boundary='fill', fill_value=0.,
                  nan_treatment='interpolate', normalize_kernel=True,
                  normalization_zero_tol=1e-8,
                  preserve_nan=False, mask=None, crop=True, return_fft=False,
-                 fft_pad=None, psf_pad=None, min_wt=0.0, allow_huge=False,
+                 fft_pad=None, psf_pad=None, dealias=None, min_wt=0.0, allow_huge=False,
```

Comment:
Can we move the new keyword to the end of the signature, in case some people are calling this by passing keyword as positional?

Reply:
Done in 9fd8242